docs/contributing/setup.md

```diff
@@ -51,8 +51,29 @@ and [here](https://github.com/electron-userland/electron-builder/issues/993#issu
 ```shellsession
 $ sudo ln -s `find /usr/lib64/ -type f -name "libbz2.so.1*"` /usr/lib64/libbz2.so.1.0
 $ sudo ln -s `find /usr/lib64/ -type f -name "libreadline.so.7.0"` /usr/lib64/libreadline.so.6
+
+### Ubuntu 16.04
```

Comment:
The code code just before this title is not closed. You have removed the closing <code>\`\`\`</code>.

Reply:
Good catch. Has been fixed in ded03ea2863b91c97a2eaee07295ace38f08c22d

Reply:
This was fixed in #3160